go/private/tools/path.bzl

```diff
@@ -75,7 +75,7 @@ Found {} in
     seen_libs[golib.importpath] = golib
     package_files = []
     prefix = "src/" + golib.importpath + "/"
-    for src in golib.srcs:
+    for src in as_iterable(golib.srcs) + tuple(as_iterable(golib.runfiles.files)):
```

Comment:
sure thing

Reply:
the thing is that i need the importpath to put them in the fake gopath properly, so keeping the archives is rather useful

Reply:
Hmm, makes sense. There needs to be some association between the import path and the data files, but we need to avoid placing `runfiles` in `GoArchiveData` to avoid a blowup in data structure size and time to iterate.

Instead of adding `runfiles` to `GoArchiveData`, I'd suggest adding a `data` field, which will be a tuple of data files just for that archive. This will be essentially the same format as `srcs`. You can get this tuple in `emit_archive` by iterating `source.runfiles` before it's merged.